gridded_etl_tools/utils/extractor.py

```diff
@@ -10,75 +10,64 @@
 if TYPE_CHECKING:  # pragma NO COVER
     from .. import dataset_manager
 
+from abc import ABC, abstractmethod
+import asyncio
+import nest_asyncio
 import pathlib
 import typing
 import ftplib
 import re
 import time
-import multiprocessing
 import logging
 
 
 log = logging.getLogger("extraction_logs")
```

Comment:
will this get passed through to Prefect as things stand or do we need to add it to Prefect’s loggers?